pkg/machinery/constants/constants.go

```diff
@@ -13,7 +13,7 @@ import (
 
 const (
 	// DefaultKernelVersion is the default Linux kernel version.
-	DefaultKernelVersion = "5.10.29-talos"
+	DefaultKernelVersion = "5.10.38-talos"
```

Comment:
Should it be added to release.toml now or later? What is our process for that?

Reply:
And should we update it to 5.10.42?

Reply:
we can update it later